package.json

```diff
@@ -4,7 +4,9 @@
   "description": "openaps oref0 reference implementation of the reference design",
   "scripts": {
     "test": "make test",
-    "global-install": "npm install && sudo npm install -g && sudo npm link && sudo npm link oref0"
+    "deprecated-global-install" : "npm install && sudo npm install -g && sudo npm link && sudo npm link oref0",
+    "global-install": "Error: global-install is deprecated. Please use 'npm install -g oref0' for a packaged version, or 'npm install -g .' for a git checkout. See https://github.com/openaps/oref0/pull/302",
```

Comment:
@scottleibrand 
According to https://docs.npmjs.com/cli/link I believe your right. But why should we a normal user (non developer) link the files in the package? The current dev branch always installs the packaged version, see https://github.com/openaps/oref0/blob/dev/bin/oref0-setup.sh#L256 so we should not need a git clone.

Unfortuately there are quite still 10 references to $HOME/src/oref0. Maybe we should stop using (relative) links to files in the package in the script and use `curl` or `wget` to get those files from github.com.

Maybe @bewest knows a smart way that's easy for both the developer (so that you don't have to re-run every time you change something), and a standard deployment way.

I think git cloning and then using a packaged version is a confusing way of deployment.